config/routes.rb

```diff
@@ -1,6 +1,20 @@
 Rails.application.routes.draw do
+  get 'sessions/new'
+  get 'sessions/create'
+  get 'sessions/destroy'
+  resources :users
+  root 'home#index'
```

Comment:
i dunno about adding stuff thats exactly what you have before but with a different name